package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Restaurant-page",
+  "version": "1.0.0",
+  "description": "Practicing DOM manipulation by dynamically rendering a simple restaurant homepage",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack --config webpack.config.js",
```

Comment:
- [ ] Include "dev" script here also in this format: "dev": "webpack-dev-server --config ./webpack.config.js",